### PR TITLE
[tests-only] [full-ci] Test for activity filters

### PR DESCRIPTION
**Pull request by Salipa-Gurung**
## Related Issue
- Part of https://github.com/owncloud/ocis/issues/9543
- https://github.com/owncloud/ocis/issues/9850
- https://github.com/owncloud/ocis/issues/9880

## How Has This Been Tested?
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/TestHelpers/GraphHelper.php [hunk 1]
```diff
@@ -2334,11 +2336,17 @@ public static function getActivities(
 		string $requestId,
 		string $user,
 		string $password,
-		string $resourceId
+		string $resourceId,
+		?string $filterType = "",
+		?string $filter = null
```
Comment (saw-jan) [inline tests/TestHelpers/GraphHelper.php:2341]:
```suggestion
		?string $filterParams = []    // e.g.: ["depth" => 1]
```

File: tests/TestHelpers/GraphHelper.php [hunk 2]
```diff
@@ -2334,11 +2336,17 @@ public static function getActivities(
...
 	): ResponseInterface {
 		// 'kql=itemId' filter is required for the current implementation but it might change in future
 		// See: https://github.com/owncloud/ocis/issues/9194
-		$fullUrl = self::getBetaFullUrl($baseUrl, "extensions/org.libregraph/activities?kql=itemid%3A$resourceId");
+		if ($filter !== null) {
+			$fullUrl = self::getBetaFullUrl($baseUrl, "extensions/org.libregraph/activities?kql=itemid%3A$resourceId+AND+$filterType%3A$filter");
+		} else {
+			$fullUrl = self::getBetaFullUrl($baseUrl, "extensions/org.libregraph/activities?kql=itemid%3A$resourceId");
+		}
```
Comment (saw-jan) [inline tests/TestHelpers/GraphHelper.php:2349]:
```suggestion
		$fullUrl = self::getBetaFullUrl($baseUrl, "extensions/org.libregraph/activities?kql=itemid%3A$resourceId");
		if (!\empty($filterParams)) {
			// loop through the params and build the query string
		}
```

**Review by SagarGi — COMMENTED** [review]
other LGTM :+1:

**Review by saw-jan — APPROVED** [review]
:+1:

**Pull request merged**